### PR TITLE
2.5 supply points are given everytime a marine spawn

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Ttile. Supply points start at 0. Doesn't work on respawn

## Why It's Good For The Game

Takes into account the serv pop. Overall, give more supply points in high pop, but since SG was removed i see this as ok balance wise

## Changelog
:cl:
balance: Every marine spawning for the first time will provide req 2.5 points
/:cl:

### CODE REVIEW

**Comment by SpaceLoveSs13** [comment]
Does this give point when marines respawn too? or just lobby spawns?

**Comment by BraveMole** [comment]
> Does this give point when marines respawn too? or just lobby spawns?

Just lobby spawn

**Comment by Swagile** [comment]
does this work for latespawns?

for example i don't ready up sometimes cause i want to see if someone is taking a critical role or not (for example, i do this during low pop so that i can take FC or SL if no one does)

**Comment by BraveMole** [comment]
> does this work for latespawns?
> 
> for example i don't ready up sometimes cause i want to see if someone is taking a critical role or not (for example, i do this during low pop so that i can take FC or SL if no one does)

yes

**Comment by SplinterGP** [comment]
Giving this a TM, might be too much, maybe not, who knows

File: code/__DEFINES/mode.dm [hunk 1]
```diff
@@ -112,6 +112,8 @@
 #define COOLDOWN_COMM_MESSAGE 	1 MINUTES
 #define COOLDOWN_COMM_CENTRAL 	30 SECONDS
 
+#define SUPPLY_POINT_MARINE_SPAWN 5
```
Comment (SplinterGP) [inline code/__DEFINES/mode.dm:115]:
```suggestion
#define SUPPLY_POINT_MARINE_SPAWN 5
```
Actually i believe this should be gamemode dependant, just add a var to gamemode type for this

**Review by SplinterGP — CHANGES_REQUESTED** [review]

File: code/__DEFINES/mode.dm (hunk 1 above)
Comment (BraveMole) [reply code/__DEFINES/mode.dm:115]: The only gamemode that uses req point is distress, no crash...?

**Review by SplinterGP — APPROVED** [review]
code ok

File: code/controllers/subsystem/points.dm [hunk 2]
```diff
@@ -13,7 +13,7 @@ SUBSYSTEM_DEF(points)
 	wait = 10 SECONDS
 
 	var/dropship_points = 0
-	var/supply_points = 120
+	var/supply_points
```
Comment (SplinterGP) [inline code/controllers/subsystem/points.dm:16]: actually
```suggestion
	var/supply_points = 0
```

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Review by SplinterGP — APPROVED** [review]

**Comment by SplinterGP** [comment]
Apparently this is alright @TiviPlus @BraveMole ? i'll merge tomorrow if is

**Comment by BraveMole** [comment]
Yeah i think it's fine, especially since free sg kits are back

**Pull request merged**